packages/Vendor/src/ValueObjects/VendorId.php

```
@@ -0,0 +1,43 @@
+{
+    private const ULID_LENGTH = 26;
+    private const ULID_PATTERN = '/^[0-9A-HJKMNP-TV-Z]{26}$/';
+
+    private string $value;
+
+    public function __construct(string $value)
+    {
+        $normalized = strtoupper(trim($value));
+
+        if ($normalized === '') {
+            throw new \InvalidArgumentException('Vendor ID cannot be empty.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '19', column '23'). (undefined)

(MissingImport)